meta-mender-variscite/recipes-bsp/u-boot/patches/imx8mm_var_dart_defconfig.mender.patch

```diff
@@ -0,0 +1,54 @@
+From 0987b38e64f4a196424c5884e2ac64274e53817d Mon Sep 17 00:00:00 2001
+From: Esteban AP <esteban.aguero@northern.tech>
+Date: Wed, 5 Apr 2023 17:19:58 +0000
+Subject: [PATCH] conf for imx8mm_var_dart board for mender support
+
+Signed-off-by: Esteban AP <esteban.aguero@northern.tech>
+---
+ configs/imx8mm_var_dart_defconfig | 33 +++++++++++++++++++++++++++++++
+ 1 file changed, 33 insertions(+)
+
+diff --git a/configs/imx8mm_var_dart_defconfig b/configs/imx8mm_var_dart_defconfig
+index 599eaf5c4e..c3e26cc38d 100644
+--- a/configs/imx8mm_var_dart_defconfig
++++ b/configs/imx8mm_var_dart_defconfig
+@@ -154,3 +154,36 @@ CONFIG_EFI_IGNORE_OSINDICATIONS=y
+ CONFIG_EFI_CAPSULE_FIRMWARE_RAW=y
+ CONFIG_EFI_SECURE_BOOT=y
+ # CONFIG_OPTEE_LIB is not set
++
++# Mender
```

Comment:
That looks non-defconfig-ish. Was it actually created by ´savedefconfig´, or is is manually created? If manually, better follow the canonical way.